index.native.js

```diff
@@ -0,0 +1,34 @@
+import Breakdancer from './';
+import { Dimensions } from 'react-native';
+
+export default class NativeDancer extends Breakdancer {
+  /**
+   * Return the current view port.
+   *
+   * @returns {Object} viewport
+   * @public
+   */
+  viewport () {
+    return Dimensions.get('window');
+  }
+
+  /**
+   * Lookup the view port width.
+   *
+   * @returns {Number} Current width.
+   * @public
+   */
+  width () {
+    return Dimensions.get('window').width;
```

Comment:
But that would introduce an extra function call as overhead, which is not something you really want.